templates/workspaces/base/terraform/azure-monitor/azure-monitor.tf

```diff
@@ -44,18 +44,41 @@ resource "azurerm_log_analytics_linked_storage_account" "workspace_storage_custo
   storage_account_ids   = [azurerm_storage_account.app_insights.id]
 }
 
-resource "azurerm_monitor_private_link_scope" "workspace" {
-  name                = "ampls-${var.tre_id}-ws-${local.short_workspace_id}"
-  resource_group_name = var.resource_group_name
-  tags                = var.tre_workspace_tags
+# TODO: Swith to azurerm once the issue is resolved
+# resource "azurerm_monitor_private_link_scope" "workspace" {
+#   name                = "ampls-${var.tre_id}-ws-${local.short_workspace_id}"
+#   resource_group_name = var.resource_group_name
+#   tags                = var.tre_workspace_tags
+
+#   lifecycle { ignore_changes = [tags] }
+# }
+
+resource "azapi_resource" "pls_workspace" {
+  type      = "microsoft.insights/privateLinkScopes@2021-07-01-preview"
+  name      = "ampls-${var.tre_id}-ws-${local.short_workspace_id}"
+  parent_id = var.resource_group_id
+  location  = "global"
+  tags      = var.tre_workspace_tags
+
+  body = jsonencode({
+    properties = {
+      accessModeSettings = {
+        ingestionAccessMode = "PrivateOnly"
+        queryAccessMode     = "PrivateOnly"
```

Comment:
I don't think this line is right

Reply:
I verified using resource explorer what is being deployed at present and replicated.

What do you think it should be?